### ISSUE TITLE
confusion about how to report BF results

### ISSUE THREAD

**Issue by metamorphose42** [jasp-stats/jasp-desktop#2215]
Hi,

I am trying to figure out how to best report my results. In general, I look at both BF10 and BF01. Often my approach is to look at BF10, if this is < 1, I look at the BF01, which usually provides stronger evidence.

For example, I currently have a simple design with two factors. For all four factors incl effects the BF10 is around or below 1. When I look at the BF01, the best model providing the strongest evidence (BF01=12) is the model incl both factors and their interaction.
On the website, the explanation I find is that the "BF10 is the column that lists the BF for each model against the null". So then the BF01 lists the BF for the null against each model, correct?
So to report my results, I would say that there is strong evidence for the null model when compared to the model with incl both factors and their interaction. Which in layman terms implies I have strong evidence that there are no behavioural effects in my experiment, correct?
Can I also mention the other models, or is this redundant, and just stick to the best model?

For another example, where I ran a lot of t tests, I want to report both BF10 and BF01 for each result. Is this unnecessary or redundant? (This is for my PhD thesis, not a paper, so I want to provide the full picture).

Maybe I am understanding it still wrong. But if BF10 provides evidence for the alternative when compared to the null, then similarly I have to report the BF01 when I want to show that there is evidence of the null against the alternative, or?

From the workshop I remembered that reporting just the "Effects" analyses might be a short cut, so I wanted to work with my actual model comparisons.

Thanks!

Best,
Clarisse

**Comment by EJWagenmakers** [comment]
Hi Clarisse,

Thanks for your message. I'll answer it here, but for these kinds of interpretational issues the JASP forum (http://forum.cogsci.nl/index.php?p=/categories/jasp-bayesfactor) is more appropriate. 

Anyway, I think your explanation is correct, but I would have to see a screenshot of the JASP output table to be sure. Not that BF10 = 1/BF01. What I usually do is select "compare to best model" -- now the first row shows the model that is supported most. All the BF10's will be smaller than 1, so it is handy to tick "BF01" -- now you will see how more likely the data are under the best model compared to each of the other models.
E.J.

**Comment by AlexanderLyNL** [comment]
I think that this issue is solved. If not, please re-open.
